documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -142,21 +151,28 @@
 - [ ] Merge the PR
    - Merging the PR will trigger a CI run that will build assets, attach the assets to the GitHub release, publish the GitHub release, and create the corresponding git tag.
  - [ ] Update `🚢 Estimated shipping date` table
- - [ ] Comment on this issue announcing the RC
+ - [ ] Comment on this issue announcing the release:
     - Link to issue comment:
 
 #### Testing for {{$rc}}
 > [!NOTE]
 > Link to any special steps for testing releases beyond ensuring CI is green.  Steps can be inlined here or tracked elsewhere.
 
 <!--{{end}}-->
+</details>
 
 ## ➡ Post-Release
+<details>
+  <summary open>Section</summary>
 
 - [ ] Open a PR against `master` cherry-picking the CHANGELOG commits from the `release/v{{.Tag}}` branch. Title it `chore(release): cherry-pick v{{.Tag}} changelog back to master`
    - Link to PR:
    - Assuming we followed [the process of merging changes into `master` first before backporting to the release branch](https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md#branch-and-tag-strategy), the only changes should be CHANGELOG updates.
 - [ ] Finish updating/merging the [RELEASE_ISSUE_TEMPLATE.md](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md) PR from `Before RC1` with any improvements determined from this latest release iteration.
+- [ ] Review and approve the auto-generated PR in [lotus-docs](https://github.com/filecoin-project/lotus-docs/pulls) that updates the latest Lotus version information.
+- [ ] Review and approve the auto-generated PR in [filecoin-docs](https://github.com/filecoin-project/filecoin-docs/pulls) that updates the latest Lotus version information.
+
```

Comment:
It is uncertain whether we should include this at this time, as I am unsure if the auto-generation feature is currently implemented for Filecoin-docs.

Reply:
Agreed that this comment is inaccurate until https://github.com/filecoin-project/filecoin-docs/issues/2348 is handled.

Should we instead say:

```suggestion
- [ ] Update the Lotus version referenced in [filecoin-docs](https://github.com/filecoin-project/filecoin-docs/pulls) ([example](https://github.com/filecoin-project/filecoin-docs/pull/####)).
```

Or maybe we don't do this because we don't have any version stuff hardcoded in Filecoin docs?  In rereading https://docs.filecoin.io/nodes/lite-nodes/spin-up-a-lite-node#pre-build, I see that it just uses a version number as an example but has code for fetching the latest version.  I didn't search elsewhere through the docs but maybe we're good?

In that case, just remove this line?

Reply:
After a quick search on docs.filecoin.io, I didn't find any references to Lotus versions. Therefore, I'll remove this line for now. We can reintroduce it later if we determine it's necessary.